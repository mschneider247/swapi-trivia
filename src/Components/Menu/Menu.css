.menu {
  display: flex;
}

h1 {
  font-size: 14rem;
  margin: 0;
}
