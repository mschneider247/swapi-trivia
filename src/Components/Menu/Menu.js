import React from 'react';
import './Menu.css';
import { Link } from 'react-router-dom';

const Menu = ( props ) => {

  return (
    <nav>
      <h1><span className='yellow-text'>SW</span>API</h1>
      <p>FAVORITES</p>
      <Link to='/'><p onClick={() => props.removeUserState()}>LOGOUT</p></Link>
    </nav>
  )
}

export default Menu;
