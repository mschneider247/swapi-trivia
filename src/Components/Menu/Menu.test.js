import React from 'react';
import { shallow } from 'enzyme';
import Menu from './Menu';

describe('Menu', () => {
  
  beforeEach(() => {
  })
  
  it('Should match the snapshot', () => {
    const wrapper = shallow( <Menu /> );
    expect(wrapper).toMatchSnapshot();
  })
})