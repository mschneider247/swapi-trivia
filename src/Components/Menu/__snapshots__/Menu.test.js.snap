// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu Should match the snapshot 1`] = `
<nav
  className="menu"
>
  <Route
    path="/movies"
    render={[Function]}
  />
  <Route
    path="/favorites"
    render={[Function]}
  />
  <div
    className="favandlog"
  >
    <Route
      path="/movies"
      render={[Function]}
    />
    <Route
      path="/favorites"
      render={[Function]}
    />
    <Route
      path="/movies"
      render={[Function]}
    />
    <Route
      path="/favorites"
      render={[Function]}
    />
  </div>
</nav>
`;
